packages/cli/src/cli/cogify/action.make.cog.pr.ts

```diff
@@ -13,15 +13,14 @@ export enum Category {
 
 export interface CategorySetting {
   minZoom?: number;
-  disabled?: boolean;
 }
 
 export const DefaultCategorySetting: Record<Category, CategorySetting> = {
   [Category.Urban]: { minZoom: 14 },
   [Category.Rural]: { minZoom: 13 },
   [Category.Satellite]: { minZoom: 5 },
-  [Category.Event]: { minZoom: 32 },
-  [Category.Other]: { minZoom: 32 },
+  [Category.Event]: {},
```

Comment:
we should add a `addToAerial: false` here ?

Reply:
Added.